addons/storyshots/stories/required_with_context/Button.stories.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line
 import { action } from '@storybook/addon-actions'; // eslint-disable-line
 
-import Button from './Button';
+import Button from '@storybook/components/dist/demo/Button';
```

Comment:
@tmeasday @ndelangen my understanding (possibly wrong) was that `demo` is just one type of component, and that we are going to move all the storybook components into here. If that's the case, I think importing from a sub-path makes sense.